.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Mint release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Release tag
+        required: true
+        default: 0.9.0rc1
+      DOI:
+        description: Reserved DOI from Zenodo
```

Comment:
Neither for the first time nor the last time, I am filled with regret that there is no emoji for a persistent identifier. 😐